odoo/addons/base/tests/test_base.py

```diff
@@ -52,13 +64,275 @@ def test_05_safe_eval_forbiddon(self):
             safe_eval('open("/etc/passwd","r")')
 
         # no forbidden opcodes
-        with self.assertRaises(ValueError):
+        with self.assertRaises(SyntaxError):
             safe_eval("import odoo", mode="exec")
 
         # no dunder
         with self.assertRaises(NameError):
             safe_eval("self.__name__", {'self': self}, mode="exec")
 
+    def test_05_call_checker_injection(self):
+        """Test that the call_checker is injected into the code"""
```

Comment:
https://peps.python.org/pep-0257/

I use that to check my docstring: https://pypi.org/project/pydocstyle/

(recheck the other docstring of that pr)

```suggestion
        """Test that the call_checker is injected into the code."""